wiki/en/Running-a-Server.md

```diff
@@ -193,38 +180,14 @@ Now you can run these with the `systemctl` command, for example:
 
 You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
 
-### Making a Server status page
-
-With the `-m` command line argument, Server statistics can be generated to be put on a web page.
-
-Here is an example php script using the Server status file to display the current Server status on a html page (assuming the following command line argument to be used: `-m /var/www/stat1.dat`):
-
-~~~
-<?php
-function loadserverstat ( $statfilename )
-{
-   $datei = fopen ( $statfilename, "r" );
-   while ( !feof ( $datei ) )
-   {
-          $buffer = fgets ( $datei, 4096 );
-          echo $buffer;
-   }
-   fclose($datei);
-}
-?>
-<?php loadserverstat ( "stat1.dat" ); ?>
-~~~
 
+## Servers on the desktop
```

Comment:
As noted, I'm not particularly happy with the split.

Reply:
I think this should be part of another PR.

Reply:
Covered by https://github.com/jamulussoftware/jamuluswebsite/issues/729